manim/mobject/coordinate_systems.py

```diff
@@ -87,14 +120,33 @@ def get_axis_labels(self, x_label_tex="x", y_label_tex="y"):
         )
         return self.axis_labels
 
+    def get_line_from_axis_to_point(
```

Comment:
is this not the same like `get_vertical_line_to_graph`?
In case that not, maybe you can add an example here:
https://manimce--1161.org.readthedocs.build/en/1161/examples.html#sinandcosfunctionplot
so one can see in action what the difference between these two is

Reply:
I compared the two and I think they're a bit different: `axis_to_point` takes in a point anywhere on the graph, whereas `line_to_graph` is specifically linked to a predefined function. They're definitely at least similar. That example is already a little busy, so adding another line doesn't seem like the best idea though.

Reply:
Okay, nice, that makes sense.
Then we can e.g. add it here:
https://manimce--1161.org.readthedocs.build/en/1161/examples.html#graphareaplot

Reply:
I just realized that it actually won't work because this method is in `CoordinateSystem` and `GraphScene` does not rely on `CoordinateSystem` at all. The documentation changes should probably be saved when I port the examples to `Axes`.